third_party/BUILD

```diff
@@ -605,6 +605,39 @@ filegroup(
     ],
 )
 
+pkg_tar(
```

Comment:
These pkg_tar should have a comment to explain why they're needed in Bazel.

Reply:
Done.